[0m18:02:03.857617 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A4CB17E720>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A4CB17E900>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A4CB17EAB0>]}


============================== 18:02:03.885328 | 7f282773-fff3-4902-b52a-f6fb6d12da2c ==============================
[0m18:02:03.885328 [info ] [MainThread]: Running with dbt=1.7.10
[0m18:02:03.887249 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\cmotte\\.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m18:02:03.892276 [info ] [MainThread]: dbt version: 1.7.10
[0m18:02:03.895595 [info ] [MainThread]: python version: 3.12.2
[0m18:02:03.896094 [info ] [MainThread]: python path: c:\Users\cmotte\python_playground\.venv\Scripts\python.exe
[0m18:02:03.900984 [info ] [MainThread]: os info: Windows-10-10.0.19045-SP0
[0m18:02:03.901489 [info ] [MainThread]: Using profiles dir at C:\Users\cmotte\.dbt
[0m18:02:03.901489 [info ] [MainThread]: Using profiles.yml file at C:\Users\cmotte\.dbt\profiles.yml
[0m18:02:03.905701 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\cmotte\dbt_project\dbt_project.yml
[0m18:02:03.910513 [info ] [MainThread]: Configuration:
[0m18:02:03.911386 [info ] [MainThread]:   profiles.yml file [[31mERROR not found[0m]
[0m18:02:03.911386 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m18:02:03.915227 [info ] [MainThread]: Required dependencies:
[0m18:02:03.915227 [debug] [MainThread]: Executing "git --help"
[0m18:02:03.925604 [info ] [MainThread]:  - git [[31mERROR[0m]

[0m18:02:03.930605 [info ] [MainThread]: Connection test skipped since no profile was found
[0m18:02:03.930605 [info ] [MainThread]: [31m3 checks failed:[0m
[0m18:02:03.937946 [info ] [MainThread]: dbt looked for a profiles.yml file in C:\Users\cmotte\.dbt\profiles.yml, but did
not find one. For more information on configuring your profile, consult the
documentation:

https://docs.getdbt.com/docs/configure-your-profile


[0m18:02:03.937946 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\cmotte\dbt_project\dbt_project.yml> not found

[0m18:02:03.943169 [info ] [MainThread]: Error from git --help: Could not find command, ensure it is in the user's PATH and that the user has permissions to run it: "git"

[0m18:02:03.945160 [debug] [MainThread]: Command `dbt debug` failed at 18:02:03.945160 after 0.25 seconds
[0m18:02:03.950280 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A4CB0D0680>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A4CB09E360>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001A4CAF06A20>]}
[0m18:02:03.951427 [debug] [MainThread]: Flushing usage events
[0m18:02:03.966801 [debug] [MainThread]: Error sending anonymous usage statistics. Disabling tracking for this execution. If you wish to permanently disable tracking, see: https://docs.getdbt.com/reference/global-configs#send-anonymous-usage-stats.
[0m18:35:34.967874 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002289EA7EA80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002289EA7EB70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002289EA7EBA0>]}


============================== 18:35:34.999627 | 842ccffd-850c-4c70-9cc4-fac846ddfaf8 ==============================
[0m18:35:34.999627 [info ] [MainThread]: Running with dbt=1.7.10
[0m18:35:35.004692 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\cmotte\\.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m18:35:35.005197 [info ] [MainThread]: dbt version: 1.7.10
[0m18:35:35.005197 [info ] [MainThread]: python version: 3.12.2
[0m18:35:35.009758 [info ] [MainThread]: python path: c:\Users\cmotte\python_playground\.venv\Scripts\python.exe
[0m18:35:35.009758 [info ] [MainThread]: os info: Windows-10-10.0.19045-SP0
[0m18:35:35.013739 [info ] [MainThread]: Using profiles dir at C:\Users\cmotte\.dbt
[0m18:35:35.017708 [info ] [MainThread]: Using profiles.yml file at C:\Users\cmotte\.dbt\profiles.yml
[0m18:35:35.018210 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\cmotte\dbt_project\dbt_project.yml
[0m18:35:35.021618 [info ] [MainThread]: Configuration:
[0m18:35:35.021618 [info ] [MainThread]:   profiles.yml file [[31mERROR not found[0m]
[0m18:35:35.025685 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m18:35:35.025685 [info ] [MainThread]: Required dependencies:
[0m18:35:35.030709 [debug] [MainThread]: Executing "git --help"
[0m18:35:35.037176 [info ] [MainThread]:  - git [[31mERROR[0m]

[0m18:35:35.037176 [info ] [MainThread]: Connection test skipped since no profile was found
[0m18:35:35.042244 [info ] [MainThread]: [31m3 checks failed:[0m
[0m18:35:35.043656 [info ] [MainThread]: dbt looked for a profiles.yml file in C:\Users\cmotte\.dbt\profiles.yml, but did
not find one. For more information on configuring your profile, consult the
documentation:

https://docs.getdbt.com/docs/configure-your-profile


[0m18:35:35.043656 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\cmotte\dbt_project\dbt_project.yml> not found

[0m18:35:35.048074 [info ] [MainThread]: Error from git --help: Could not find command, ensure it is in the user's PATH and that the user has permissions to run it: "git"

[0m18:35:35.053118 [debug] [MainThread]: Command `dbt debug` failed at 18:35:35.053118 after 0.21 seconds
[0m18:35:35.053118 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002289D339DF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002289E8C5250>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002289E848770>]}
[0m18:35:35.053118 [debug] [MainThread]: Flushing usage events
[0m18:37:39.207982 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E6447A6450>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E6447A7590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E6447A6D80>]}


============================== 18:37:39.224751 | 029ab03c-ede0-45c6-a35e-c58cd446772e ==============================
[0m18:37:39.224751 [info ] [MainThread]: Running with dbt=1.7.10
[0m18:37:39.225920 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\cmotte\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt run', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m18:37:39.227563 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\cmotte\dbt_project\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m18:37:39.229919 [debug] [MainThread]: Command `dbt run` failed at 18:37:39.229919 after 0.23 seconds
[0m18:37:39.230923 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E64477E750>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E64441E8A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E6446209B0>]}
[0m18:37:39.231920 [debug] [MainThread]: Flushing usage events
[0m09:56:51.717246 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DECA1A1550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DECA1A2330>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DECA1A0E30>]}


============================== 09:56:51.730349 | f0fd097f-3a49-4ac0-af0f-9fa2932e5bed ==============================
[0m09:56:51.730349 [info ] [MainThread]: Running with dbt=1.7.10
[0m09:56:51.732227 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\cmotte\\.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m09:56:51.732730 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\cmotte\dbt_project\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m09:56:51.735759 [debug] [MainThread]: Command `dbt run` failed at 09:56:51.734760 after 0.17 seconds
[0m09:56:51.736760 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DEC9BED730>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DECA0231D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DEC9EC0170>]}
[0m09:56:51.737846 [debug] [MainThread]: Flushing usage events
[0m18:40:20.244660 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000294C00A6930>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000294BFE326F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000294C00A6420>]}


============================== 18:40:20.269678 | ca4e7667-325e-4813-9664-0b1b22c21bd6 ==============================
[0m18:40:20.269678 [info ] [MainThread]: Running with dbt=1.7.10
[0m18:40:20.273913 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\cmotte\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt test', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m18:40:20.273913 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\cmotte\dbt_project\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m18:40:20.278885 [debug] [MainThread]: Command `dbt test` failed at 18:40:20.278885 after 0.19 seconds
[0m18:40:20.283981 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000294BFD9B5F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000294BFE326F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000294BF3FD070>]}
[0m18:40:20.285796 [debug] [MainThread]: Flushing usage events
[0m18:40:20.302268 [debug] [MainThread]: Error sending anonymous usage statistics. Disabling tracking for this execution. If you wish to permanently disable tracking, see: https://docs.getdbt.com/reference/global-configs#send-anonymous-usage-stats.
